File: pinyougou-content-interface/src/main/java/com/pinyougou/content/service/ContentService.java
package com.pinyougou.content.service;

import com.pinyougou.pojo.TbContent;

import java.util.List;

public interface ContentService {
    /**
     * @Author: ZhangYu
     * @Date:2019-11-18
     * @Description: java学习
     * @Question: 查询所有的广告
     */
    List<TbContent> findAll();

    /**
     * @Author: ZhangYu
     * @Date:2019-11-18
     * @Description: java学习
     * @Question: 添加广告
     */
    void add(TbContent content);

    /**
     * @Author: ZhangYu
     * @Date:2019-11-18
     * @Description: java学习
     * @Question: 修改广告
     */
    void update(TbContent content);

    /**
     * @Author: ZhangYu
     * @Date:2019-11-18
     * @Description: java学习
     * @Question: 删除选中的广告
     */
    void delete(Long[] ids);

    /**
     * @Author: ZhangYu
     * @Date:2019-11-18
     * @Description: java学习
     * @Question: 查询一个广告实例
     */
    TbContent findOne(Long id);

    /**
     * @Author: ZhangYu
     * @Date:2019-11-18
     * @Description: java学习
     * @Question: 根据分类id查询广告
     */
    List<TbContent> findContentByCategoryId(Long categoryId);
}

File: pinyougou-manager-web/src/main/webapp/js/service/contentService.js
app.service("contentService",function ($http) {

    //查询所有的广告信息
    this.findAll = ()=>{
       return $http.get("../content/findAll.do");
    };

    //保存广告信息
    this.save = (url,entity)=>{
        return $http.post(url,entity);
    };
    //查询一个广告实例
    this.findOne = (id)=>{
        return $http.get("../content/findOne.do?id="+id);
    };
    //删除广告
    this.delete = (ids)=>{
        return $http.get("../content/delete.do?ids="+ids);
    };
});